tools/clang/lib/AST/ASTDumper.cpp

```diff
@@ -118,6 +118,8 @@ namespace  {
 
     bool ShowColors;
 
+    bool DumpImplicitTopLevelDecls = true; // HLSL Change
```

Comment:
Why initialize to true here?

Reply:
So that the default behavior of the dumper is to always dump top-level decls. Then it only ends up false if you explicitly specify false.

Reply:
I see.
That sounds reasonable.